app/views/static_pages/home.html.erb

```diff
@@ -1,9 +1,9 @@
 <% content_for :head do %>
-  <meta property="og:title" content="Debt Collective - Fundraising" />
+  <meta property="og:title" content="Debt Collective - Membership" />
   <meta property="og:description" content="Organizing for big change requires resources. Becoming a dues- paying member is not charity. Instead, your support is an act of ongoing solidarity with other people in debt." />
   <meta property="og:type" content="text/html" />
-  <meta property="og:url" content="https://fundraising.debtcollective.org" />
-  <meta property="og:image" content="https://fundraising.debtcollective.org/assets/logo-small.png" />
+  <meta property="og:url" content="https://Membership.debtcollective.org" />
```

Comment:
@castrolem this should be membership.debtcollective.org right? Also, I think is better we ask Rails for the host instead of adding it ourselves, will work for different envs

Reply:
changed